_layouts/default.html

```diff
@@ -8,6 +8,7 @@
     <meta name="keywords" content="APIs, SDKs, Mobile development, LiveEngage, Agent workspace, Tag, JSON examples, Click to message, Click to chat, Chat window, Skills, Agent Groups, iOS, Android, Mobile devices, Integrations, Live chat, Agent widget, Reporting, Analytics">
     <meta name="author" content="LivePerson Inc.">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/docsearch.js/2/docsearch.min.css">
+    {% if page.noindex == "true" %} <meta name="robots" content="noindex" >{% endif %}
```

Comment:
yap, nice :)

Reply:
{% if page.noindex %} is better and will work, because all values in liquid are truthy, except for false and nil.

Reply:
{% if page.noindex} <meta name="robots" content="noindex" >{% endif %}

Reply:
also please delete the space in the end.
{% if page.noindex} <meta name="robots" content="noindex">{% endif %}

Reply:
Spaces: Nice catch! Had copied this from the old template. Cleaned up.

Just `page.noindex`: Yes, good idea—but it can be confusing because that way, `noindex: false` would be… true (CMIIW). (The front matter entry needs to have a value.) So it seems clearer if we’re explicit about it.